css/style.css

```diff
@@ -0,0 +1,18 @@
+img {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
+
+.title_paragraph {
+  display: block;
+  text-align: center;
+  background-color: rgb(233, 225, 225);
+}
+
+#tribute-info {
+  display: block;
+  text-align: center;
+  background-color: rgb(233, 225, 225);
+}
```

Comment:
It is better to use flexbox and grid for layout of your Page it will make your website looks more good and responsive

Reply:
Good work giving reason why you are making a suggestion, it is a good review practice to show the `why` behind your suggestion. 🤝 However, as mentioned above...
- [ ] Please consider the comment above about the use of polite words ⬆️